fastlane/Helpers/Version.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  Version.swift
+//  FastlaneRunner
+//
+//  Created by Khanh on 26/09/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+enum Version {
+    
+    // MARK: - Getting
+    
+    static var versionNumber: String {
+        get {
+            FastlaneRunner.getVersionNumber(
+                xcodeproj: .userDefined(Constant.projectPath),
+                target: .userDefined(Constant.projectName)
+            )
+        }
+        
+        set {
+            incrementVersionNumber(
+                versionNumber: .userDefined(newValue),
+                xcodeproj: .userDefined(Constant.projectPath)
+            )
+        }
+    }
+    
+    static var buildNumber: String {
+        get {
+            FastlaneRunner.getBuildNumber(xcodeproj: .userDefined(Constant.projectPath))
+        }
+
+        set {
+            incrementBuildNumber(
+                buildNumber: .userDefined(newValue),
+                xcodeproj: .userDefined(Constant.projectPath)
+            )
+        }
+    }
+
+    static func getVersionAndBuildNumber() -> String {
+        "\(getVersionNumber()) (Build: \(getBuildNumber())"
```

Comment:
Don't need to use the function from fastlane in this, it might occur issue.
```suggestion
        "\(versionNumber) (Build: \(buildNumber)"
```

Reply:
I updated at https://github.com/nimblehq/ios-templates/pull/362/commits/f8a611f1ce2582c43fd34ff04f1da3b472c2b63b 🙏 .